scripts/spack/specs.json

```diff
@@ -17,7 +17,7 @@
     [ "clang@9.0.0~cpp14+devtools+mfem",
       "clang@10.0.0~cpp14+devtools+mfem",
       "gcc@8.1.0~cpp14+devtools+mfem",
-      "gcc@8.1_no_fortran~cpp14~fortran+devtools+mfem",
+      "gcc@8.1_no_fortran~cpp14~fortran+devtools~mfem",
```

Comment:
Note: I explicitly disabled `mfem` in the configuration that did not have a Fortran compiler.

A different approach would be to use a serial mfem in this case, or to figure out how to build hypre without when we don't have a Fortran compiler.